src/components/cluster/cluster.type.ts

```diff
@@ -27,34 +65,45 @@ export enum ClusterComponentStatus {
     ACD_SUCCESS = 'ACD_SUCCESS',
 }
 
-export type ClusterComponentStatusType = keyof typeof ClusterComponentStatus;
+export type ClusterComponentStatusType = keyof typeof ClusterComponentStatus
 
-export type ClusterInstallStage = -1 | 0 | 1 | 2 | 3;
+export type ClusterInstallStage = -1 | 0 | 1 | 2 | 3
 
 export interface ClusterComponentType {
-    name: string;
-    appId: number;
-    installedAppId: number;
-    envId: number;
-    envName: string;
-    status: ClusterComponentStatusType;
+    name: string
+    appId: number
+    installedAppId: number
+    envId: number
+    envName: string
+    status: ClusterComponentStatusType
 }
 
 export interface ClusterComponentModalProps {
-    agentInstallationStage: ClusterInstallStage;
-    components: ClusterComponentType[] | null;
-    environmentName: string;
-    redirectToChartDeployment: (appId, envId) => void;
-    callRetryClusterInstall: () => void;
-    close: (event) => void;
+    agentInstallationStage: ClusterInstallStage
+    components: ClusterComponentType[] | null
+    environmentName: string
+    redirectToChartDeployment: (appId, envId) => void
+    callRetryClusterInstall: () => void
+    close: (event) => void
 }
 
 export interface ClusterInstallStatusProps {
-    agentInstallationStage: ClusterInstallStage;
-    envName: string | undefined;
-    onClick: (...args) => void;
+    agentInstallationStage: ClusterInstallStage
+    envName: string | undefined
+    onClick: (...args) => void
+}
+
+export interface ConfigCluster {
+    bearerToken: string
+    cert_auth_data: string
```

Comment:
Use camelCase

Reply:
This case is expeceted in payload as we are using the api directly to connect to kubernetes.